web-local/src/lib/components/workspace/explore/filters/Filters.svelte

```diff
@@ -107,31 +114,31 @@ The main feature-set component for dashboard filters
 
   function clearAllFilters() {
     if (hasFilters) {
-      metricsExplorerStore.clearFilters(metricsDefId);
+      metricsExplorerStore.clearFilters(metricViewName);
     }
   }
 
   /** prune the values and prepare for for templating */
   let currentDimensionFilters = [];
   $: if (includeValues && excludeValues) {
     const dimensionIdMap = getMapFromArray(
-      $dimensions,
-      (dimension) => dimension.id
+      dimensions,
+      (dimension) => dimension.name
     );
     const currentDimensionIncludeFilters = includeValues.map(
       (dimensionValues) => ({
-        name: getDisplayName(dimensionIdMap.get(dimensionValues.name)),
-        sqlName: dimensionIdMap.get(dimensionValues.name)?.dimensionColumn,
-        dimensionId: dimensionValues.name,
+        name: dimensionValues.name,
+        label: getDisplayName(dimensionIdMap.get(dimensionValues.name)),
+        sqlName: dimensionIdMap.get(dimensionValues.name)?.name,
         selectedValues: dimensionValues.in,
         filterType: "include",
       })
     );
     const currentDimensionExcludeFilters = excludeValues.map(
       (dimensionValues) => ({
-        name: getDisplayName(dimensionIdMap.get(dimensionValues.name)),
-        sqlName: dimensionIdMap.get(dimensionValues.name)?.dimensionColumn,
-        dimensionId: dimensionValues.name,
+        name: dimensionValues.name,
+        label: getDisplayName(dimensionIdMap.get(dimensionValues.name)),
+        sqlName: dimensionIdMap.get(dimensionValues.name)?.name,
```

Comment:
Can't we just use `dimensionValues.name` instead of using a map to get the `name`?

Reply:
We probably don't need `sqlName`

Reply:
Ya it is the same as name now.